projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -41,14 +41,14 @@
 		}
 	}
 
-	.block-editor-block-list__layout {
+	.jetpack-contact-form.block-editor-block-list__layout {
```

Comment:
We need to make sure these styles only apply to the contact form. Prior to this change they'd apply to any block that has this classname.